fuels-abigen-macro/Cargo.toml

```diff
@@ -26,4 +26,4 @@ fuels-contract = { version = "0.4.3", path = "../fuels-contract" }
 fuels-signers = { version = "0.4.3", path = "../fuels-signers", features=["test-helpers"] }
 hex = { version = "0.4.3", default-features = false, features = ["std"] }
 sha2 = "0.9.5"
-tokio = "1.15.0"
+tokio = "1.15.0"
```

Comment:
newline

If you're using vscode, turn on adding newline at end of file

Reply:
It's the rustfmt formatter it's weird